tests/components/input_datetime/test_init.py

```diff
@@ -701,7 +709,7 @@ async def test_timestamp(hass):
         assert (
             dt_util.as_local(
                 datetime.datetime.fromtimestamp(
-                    state_without_tz.attributes[ATTR_TIMESTAMP]
+                    state_without_tz.attributes[ATTR_TIMESTAMP], datetime.timezone.utc
```

Comment:
Without the enforced UTC:

`assert '2020-12-13 11:00:00' == '2020-12-13 10:00:00'`